nxcomp/src/Loop.cpp

```diff
@@ -7860,7 +7864,7 @@ int ParseEnvironmentOptions(const char *env, int force)
 
   value = strrchr(nextOpts, ':');
 
-  if (value != NULL)
+  if (value != NULL && nxdisplay_found )
```

Comment:
IMHO, nxdisplay_found the way you introduced it will always be TRUE down here. I think this whole commit is superfluous.

Reply:
Not always, no. Look at the branch with the 'force' argument.

Here's a demonstration:
```

 nxproxy-gdb  -S cups=unix://private/vda localhost:40
GNU gdb (GDB) Fedora 8.1.1-3.fc28
...
Reading symbols from /home/vadim/git/nx-libs/nxproxy/src/nxproxy...done.
(gdb) break Loop.cpp:7867
No source file named Loop.cpp.
Make breakpoint pending on future shared library load? (y or [n]) y
Breakpoint 1 (Loop.cpp:7867) pending.
(gdb) run
Starting program: /home/vadim/git/nx-libs/nxproxy/src/nxproxy -S cups=unix://private/vda localhost:40

[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".

Breakpoint 1, ParseEnvironmentOptions (env=0x7fffffffe4bb "cups=unix://private/vda", force=1) at Loop.cpp:7867

7867      if (value != NULL && nxdisplay_found )
(gdb) print nxdisplay_found
$1 = false
(gdb) bt
#0  ParseEnvironmentOptions (env=0x7fffffffe4bb "cups=unix://private/vda", force=1) at Loop.cpp:7867
#1  0x00007ffff7b0fceb in ParseCommandLineOptions (argc=4, argv=0x7fffffffe068) at Loop.cpp:8842
#2  0x00007ffff7af21cd in NXTransParseCommandLine (argc=4, argv=0x7fffffffe068) at Loop.cpp:1287
#3  0x0000000000400a06 in main (argc=4, argv=0x7fffffffe068) at Main.c:93
```